style.css

```diff
@@ -830,3 +879,181 @@ header nav.desktop a:visited {
+  box-sizing: border-box;
+  height: 100%;
+  width: 100%;
+  padding: calc(0.2 * (100vmin - 375px) + 24px);
+  backdrop-filter: blur(8px);
+}
+
+.project-details .project-card {
+  display: flex;
+  flex-direction: column;
+  align-content: flex-start;
+  box-sizing: border-box;
+  padding: 16px;
+  background-color: var(--background-color);
+}
+
+.project-details .project-image {
+  box-sizing: border-box;
+  margin: 0;
+  min-height: auto;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.project-details .toolbar {
+  position: absolute;
+  top: 2vmin;
+  right: 2vmin;
+  z-index: 5;
+  margin: 0;
+  justify-self: flex-end;
+  width: 5vmin;
+  height: 5vmin;
+  max-width: 32px;
+  max-height: 32px;
+}
+
+.project-details .toolbar button {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  object-fit: contain;
+}
+
+.project-details .project-image img {
+  width: 100%;
+  max-height: 70%;
+  border: 2px solid white;
+  border: 2px solid transparent;
+  border-radius: 5vmin;
+}
+
+.project-details .project-info {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: repeat(4, auto);
+  gap: 10px;
+}
+
+.project-details .project-title {
+  grid-column: 1/-1;
+  grid-row: 1;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 44px;
+  text-align: left;
+  color: #1d234e;
+}
+
+.project-details .project-languages {
+  grid-column: 1/-1;
+  grid-row: 2;
+  justify-content: flex-start;
+}
+
+.project-details .project-description {
+  grid-column: 1/-1;
+  grid-row: 3;
+  padding: 16px;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #344563;
+}
+
+.project-details .project-buttons {
+  grid-column: 1/-1;
+  grid-row: 4;
+  display: flex;
+  justify-content: center;
+  gap: 24px;
+  align-self: center;
+  align-content: center;
+}
+
+.project-details .project-buttons button {
+  margin: 12px;
+  display: flex;
+  gap: 5px;
+  align-items: center;
+}
+
+.project-details button span {
+  display: block;
+}
+
+.project-details .project-image button img {
+  display: block;
+}
+
+@media only screen and (min-width: 992px) {
+  .project-details .project-title {
+    grid-column: 1/-2;
+    grid-row: 1;
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 44px;
+    text-align: left;
+    color: #1d234e;
+  }
+
+  .project-details .project-languages {
+    grid-column: 1/-1;
+    grid-row: 2;
+    justify-content: flex-start;
+  }
+
+  .project-details .project-description {
+    grid-column: 1/-1;
+    grid-row: 3/4;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #344563;
+  }
+
+  .project-details .project-buttons {
+    grid-column: -2/-1;
+    grid-row: 1;
+    display: flex;
+    justify-content: center;
+    gap: 24px;
+  }
```

Comment:
- [ ] To better match the Figma design, the See Live and See Source buttons should be more to the right, aligning with the image, I imagine that justify content: right instead of center would fix this

Your design:
![image](https://user-images.githubusercontent.com/73140461/151023962-c742cb6f-244b-4fcd-afc9-0f6975a6ea4b.png)

 Figma:
![image](https://user-images.githubusercontent.com/73140461/151023793-4a5fce9f-a5f8-46a7-8df2-992943f7149c.png)

Reply:
@leolpaz,
Thank you for the great feedback.